install.sh

```diff
@@ -248,7 +248,9 @@ propose_joining_docker_group() {
     is_user_in_group "$(get_user)" docker && return 0
     is_root && return 0
     ensure_cmds_available usermod
-    [[ $override_join_docker_group == "auto" ]] && prompt_yes_no_skip 'werf needs access to the Docker daemon. Add current user to the "docker" group? (root required)' "yes" || return 0
+    if [[ $override_join_docker_group == "auto" ]]; then
+      prompt_yes_no_skip 'werf needs access to the Docker daemon. Add current user to the "docker" group? (root required)' "yes"
```

Comment:
```suggestion
      prompt_yes_no_skip 'werf needs access to the Docker daemon. Add current user to the "docker" group? (root required)' "yes" || return 0
```
`return 0` must stay, otherwise when the user answers "no" to the prompt we will not skip the current operation.

Reply:
there is no "no" answers only "yes", "skip", "abort" and all of them return status or exit status
`    case "${answer:-$def_arg}" in
      [yY] | [yY][eE][sS])
        return 0
        ;;
      [aA] | [aA][bB][oO][rR][tT])
        printf 'Aborted by the user.\n'
        exit 0
        ;;
      [sS] | [sS][kK][iI][pP])
        return 1
        ;;
      *)
        printf 'Invalid choice, please retry.\n'
        continue
        ;;
    esac
  done`
  is there a point to use || return ?